src/Components/Header.js

```diff
@@ -1,24 +1,32 @@
-import React, {Component} from 'react';
+import React, { Component } from "react";
```

Comment:
"react" should be in 'react' as we are taking this in all files like this so the pattern should be remain same. or we have to change every where

Reply:
done .....but when i use to press strl+shift+i for indentation.....it automatically convert it into "react"